Makefile

```
@@ -116,6 +117,12 @@ specs-frequency-paseo-local-debug:
 specs-frequency-paseo-local-release:
 	./scripts/generate_specs.sh 2000 frequency-paseo-local release
 
+specs-frequency-westend-local-release:
+	./scripts/generate_specs.sh 2313 frequency-westend-local release
+
+specs-frequency-westend-release:
+	./scripts/generate_specs.sh 2313 frequency-westend release
```

Doesn't matter, but usually we use 2000 for the local parachain id. If it is as easy as this, then might be worth it, otherwise nope.

```suggestion
	./scripts/generate_specs.sh 2000 frequency-westend-local release
```

---

Changed local to 2000.